src/rawParser.js

```diff
@@ -765,7 +765,6 @@ function parseString(str, pos) {
             marks.push({name: 'unicode_hex', position: pos + 3})
             pos = str.indexOf('}', pos + 3)
             marks.push({name: 'unicode_hex_end', position: pos})
-            pos++
```

Comment:
I think I found a bug in unicode hex parsing, I couldn't get this case (unicode characters delimited with braces) to work without this change. I can discuss this change offline if needs be.